homeassistant/components/prusalink/sensor.py

```diff
@@ -79,6 +79,58 @@ class PrusaLinkSensorEntityDescription(
             value_fn=lambda data: cast(float, data["telemetry"]["temp-nozzle"]),
             entity_registry_enabled_default=False,
         ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
+            key="printer.temperature.bed.actual",
+            name="Actual Heatbed Temperature",
+            native_unit_of_measurement=TEMP_CELSIUS,
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+            value_fn=lambda data: cast(float, data["temperature"]["bed"]["actual"]),
+        ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
+            key="printer.temperature.nozzle.actual",
+            name="Actual Nozzle Temperature",
+            native_unit_of_measurement=TEMP_CELSIUS,
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+            value_fn=lambda data: cast(float, data["temperature"]["tool0"]["actual"]),
+        ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
+            key="printer.temperature.bed.target",
+            name="Target Heatbed Temperature",
+            native_unit_of_measurement=TEMP_CELSIUS,
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+            value_fn=lambda data: cast(float, data["temperature"]["bed"]["target"]),
+        ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
+            key="printer.temperature.nozzle.target",
+            name="Target Nozzle Temperature",
+            native_unit_of_measurement=TEMP_CELSIUS,
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+            value_fn=lambda data: cast(float, data["temperature"]["tool0"]["target"]),
+        ),
+        PrusaLinkSensorEntityDescription[PrinterInfo](
```

Comment:
Is this interesting? I think we can disable it too by default.

Reply:
Z-height is now disabled by default.